machineconfiguration/v1/types.go

```diff
@@ -29,7 +29,10 @@ type ControllerConfig struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	// +required
+	// TODO(jkyros): inconsistent historical generation resulted in the controllerconfig CRD being
+	// generated with all fields required, while everything else was generated with optional
```

Comment:
Is this todo resolved?

Reply:
yes, everything is explicitly tagged now, comment removed